### ISSUE TITLE
question about training coco2017

### ISSUE THREAD

**Issue by locke4j** [fizyr/keras-retinanet#359]
I tried to train coco2017 myself with the default configuration, but, after about 55 epochs, the AP seems do not update (always 0.285 for 15 epochs), I don't know what can I do to improve, thank you!

**Comment by jonla1** [comment]
Check your learning rate. It decreases by a factor of 10 when there is no more improvement over some epochs. It could be that it became so small that there is no more real process

**Comment by hgaiser** [comment]
Did this resolve your issue ?
